### DISCUSSION TITLE
How to stop LED blinking when Poco's drawing commands are called

### DISCUSSION THREAD

**Discussion by harupiko** [Moddable-OpenSource/moddable#1241] [Q&A] [answered]
I'm running some of the commodetto examples on my Moddable Three. It looks like every time the drawing command, Poco.begin() and Poco.end(), is called, the LED on the moddable board blinks.

How do I stop the LED from blinking? Is there any config option to control the behavior?

**Comment by phoddie** [comment, marked as answer]
Moddable Three uses the ESP8266 which has a very limited number of GPIOs. The LED is connected to GPIO 2, which is also used as the DC pin for the display controller. Therefore, updating the display can toggle the LED.

For reference, here is the [pin diagram](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/devices/moddable-three.md#pinout) in the Moddable Three documentation and here is the [definition of the DC pin](https://github.com/Moddable-OpenSource/moddable/blob/c493992eab09b3c3e29f03ad28a575a63a261738/build/devices/esp/targets/moddable_three/manifest.json#L27) of the display controller from the Moddable Three manifest.

**Reply by harupiko** [reply]
Thank you for the answer! For the same reason, I can't turn off Moddable Two's LED while using the display.

**Reply by phoddie** [reply]
Yes, that's true. The wiring is similar.
